packages/kbn-unified-field-list/src/components/field_stats/field_top_values_bucket.tsx

```diff
@@ -79,19 +80,21 @@ const FieldTopValuesBucket: React.FC<FieldTopValuesBucketProps> = ({
         grow={1}
         css={css`
           min-width: 0;
+          word-break: break-word;
         `}
       >
         <EuiFlexGroup alignItems="stretch" gutterSize="s" responsive={false}>
           <EuiFlexItem
```

Comment:
Should the styles that perform the word breaking be moved to this `EuiFlexItem` instead for a more targeted effect?